core/arch/arm/plat-k3/platform_config.h

```diff
@@ -27,6 +27,18 @@
 #define GICD_OFFSET     0x0
 #define GICD_SIZE       0x10000
 #endif
+#if !(defined(PLATFORM_FLAVOR_am64) || defined(PLATFORM_FLAVOR_am62))
```

Comment:
Supported platforms list [here](https://optee.readthedocs.io/en/latest/general/platforms.html) only mentions `PLATFORM=k3-am65x` is supported but the check here corresponding to platform flavours `am64` and `am62` which doesn't correspond to `am65x`. Also, the commit message mentions "Tested on AM65x". So can you clarify regarding below queries?

- Which platform flavours are supported by OP-TEE?
- Which platform flavours are supposed to support this HUK feature?

Also, in case the supported platforms list is out-of-date for K3 platform flavours then do update them as well.

Reply:
Hi, the name support for the other platform flavors are being added as a part of https://github.com/OP-TEE/optee_os/pull/5299

These are the list of platform flavors that are currently supported: AM65x, J721e, AM64x and AM62x

In this, HUK is currently supported by only K3 generic devices i.e AM65x and J721e. The support for K3 Lite devices i.e AM64x and AM62x will be added subsequently. 

I'll update this codebase a bit to prevent the drivers from being built for K3 Lite devices

Reply:
Fixed them